doc/version_history.rst

```diff
@@ -6,6 +6,10 @@
 Version History
 ###############
 
+v0.10.2
```

Comment:
I think this needs to be a minor version.. even though there's little code changes, changing the module names is an API change.. if we were above version 1, this would probably mean a major release, but while in version 0 it is ok to make backward incompatible changes.